Dockerfile

```diff
@@ -31,9 +31,7 @@ FROM $RUST_IMAGE as build
 # When set, causes the proxy to be compiled in development mode.
 ARG PROXY_UNOPTIMIZED
 
-# Controls what features are enabled in the proxy. This is typically empty but
-# may be set to `mock-orig-dst` for profiling builds, or to `multithreaded` to
-# enable the multithreaded Tokio runtime.
```

Comment:
I removed this part of the comment since the `mock-orig-dst` feature is now removed and the `multithreaded` feature is enabled by default.